packages/salesforcedx-vscode-apex/package.json

```diff
@@ -1,8 +1,7 @@
 {
```

Comment:
@lcampos - What can we do here so that it follows the old format?

Reply:
I'm reverting out update on prettier to the old version in PR #561 since I didn't find a configuration that would keep our current format.

Reply:
@jkalloor0701 @vazexqi PR #561 has been merged to develop branch so please rebase and re-run `npm run reformat` to get this diffs fixed. Sorry for all the trouble.

Reply:
@jkalloor0701 my change updates the version of prettier we were using so you'll need to do a clean on your local project before running this.

Reply:
Ah, of course. Thanks